resources/qgis_global_settings.ini

```diff
@@ -1,8 +1,9 @@
 [qgis]
+formatLayerName=false
```

Comment:
Can we add comments to this file? It'd certainly help organisations customise their own version of this...